staging/src/k8s.io/apiserver/pkg/storage/selection_predicate.go

```diff
@@ -74,6 +77,7 @@ type SelectionPredicate struct {
 	Label               labels.Selector
 	Field               fields.Selector
 	GetAttrs            AttrFunc
+	IndexLabels         []string
```

Comment:
I don't think you need to *both* store them separately and prefix the name?

Reply:
The difference is that this field (or rather `IndexFields`0 is already used for a long time and those are not prefixed.
I would feel safer not merging those two in this PR - rather introducing `IndexLabels` as @shaloulcy did, potentially dropping a TODO to merge them (though that requires changing all places where it is used - there are few of them, but it's potential to break someone).

Reply:
This isn't stored anywhere, so I don't understand the risk. Anyway, my point wasn't that this shouldn't be done, just that if you do this, you don't also need to prefix the names (as stored in this object).

Reply:
> This isn't stored anywhere, so I don't understand the risk. Anyway, my point wasn't that this shouldn't be done, just that if you do this, you don't also need to prefix the names (as stored in this object).

fix it.